packages/snap-preact-components/src/components/Molecules/FacetPaletteOptions/readme.md

```diff
@@ -92,3 +92,51 @@ The `onClick` prop allows for a custom callback function for when a facet value
 ```jsx
 <FacetPaletteOptions values={paletteFacet.values} onClick={(e)=>{console.log(e)}} />
 ```
+
+
+#### colorMapping
+The `colorMapping` prop allows for custom color mapping overrides. These objects are keyed off the color label, and can take background, and a label values. The color label can be any accepted css background property value. So a color string, hash, rgb, gradiants, or an image url for more complex swatches. The label takes a string. 
+
+```jsx
+const colorMapping = {
+	'Striped': {
+		background: "url(https://mysite.com/cdn/shop/files/candy-stripe-square_small.jpg)",
+		label: "stripy"
+	}
+}
+<FacetPaletteOptions values={paletteFacet.values} colorMapping={colorMapping} />
+```
+or 
+```jsx
+const colorMapping = {
+	'rainbow': {
+		background: `linear-gradient(
+			90deg,
+			rgba(255, 0, 0, 1) 0%,
+			rgba(255, 154, 0, 1) 10%,
+			rgba(208, 222, 33, 1) 20%,
+			rgba(79, 220, 74, 1) 30%,
+			rgba(63, 218, 216, 1) 40%,
+			rgba(47, 201, 226, 1) 50%,
+			rgba(28, 127, 238, 1) 60%,
+			rgba(95, 21, 242, 1) 70%,
+			rgba(186, 12, 248, 1) 80%,
+			rgba(251, 7, 217, 1) 90%,
+			rgba(255, 0, 0, 1) 100%
+		)`,
+	}
+}
+
+<FacetPaletteOptions values={paletteFacet.values} colorMapping={colorMapping} />
```

Comment:
I like these examples, but I wonder if we need three different component usage examples, or a single usage with multiple mapping examples inside of the `colorMapping` object. /shrug